include/Graphs/SVFG.h

```diff
@@ -250,6 +250,11 @@ class SVFG : public VFG
         return nodeNum;
     }
 
+    /// Returns true if l is a delta node, i.e., may change due to
+    /// on-the-fly call graph resolution. approxCallGraph is the over-approximate
+    /// call graph built by the pre-analysis.
+    virtual bool isDeltaNode(NodeID l, PTACallGraph *approxCallGraph) const;
```

Comment:
What is a DeltaNode? Is that the dummySVFGNode in the last patch?

Reply:
Any node which may have new incoming edges due to on-the-fly callgraph resolution. So things like indirect call targets.

Reply:
Can we use a better name? Will this method be called by SFS too?

Reply:
Yes sure, I need ideas though :) (this is the name used in the paper by the way).

SFS has no need for this. Currently only VSFS uses it (it used to be in `VersionedFlowSensitive`).

Reply:
I see. Can we put this method in `VersionedFlowSensitive.cpp` then?

Reply:
Done.